Build error of 0.2.29 in windows: `missing binary operator before token "("`

Hi, after a cabal update and start to use the recently uploaded `0.2.29` version i've start to get build errors, in downstream packages and building `simple-sendfile` standalone:

```
D:\ws\haskell>cabal get simple-sendfile
Unpacking to simple-sendfile-0.2.29\

D:\ws\haskell>cd simple-sendfile-0.2.29

D:\ws\haskell\simple-sendfile-0.2.29>cabal --version
cabal-install version 3.0.0.0
compiled using version 3.0.0.0 of the Cabal library

D:\ws\haskell\simple-sendfile-0.2.29>cabal build
Resolving dependencies...
Build profile: -w ghc-8.6.5 -O1
In order, the following will be built (use -v for more details
 - streaming-commons-0.2.1.1 (lib) (requires build)
 - conduit-extra-1.3.4 (lib) (requires build)
 - simple-sendfile-0.2.29 (lib) (first run)
Starting     streaming-commons-0.2.1.1 (lib)
Building     streaming-commons-0.2.1.1 (lib)
Installing   streaming-commons-0.2.1.1 (lib)
Completed    streaming-commons-0.2.1.1 (lib)
Starting     conduit-extra-1.3.4 (lib)
Building     conduit-extra-1.3.4 (lib)
Installing   conduit-extra-1.3.4 (lib)
Completed    conduit-extra-1.3.4 (lib)
Configuring library for simple-sendfile-0.2.29..
Preprocessing library for simple-sendfile-0.2.29..
Building library for simple-sendfile-0.2.29..

Network\Sendfile\Types.hs:22:0: error:
     error: missing binary operator before token "("
     #if MIN_VERSION_unix(2,8,0)

   |
22 | #if MIN_VERSION_unix(2,8,0)
   | ^

Network\Sendfile\Types.hs:30:0: error:
     error: missing binary operator before token "("
     #if MIN_VERSION_unix(2,8,0)

   |
30 | #if MIN_VERSION_unix(2,8,0)
   | ^
`gcc.exe' failed in phase `C pre-processor'. (Exit code: 1)
```

* Tested in windows 7 and 10 with cabal-2.4.1.0 too

--- a/Network/Sendfile/Types.hs
+++ b/Network/Sendfile/Types.hs
@@ -2,11 +2,13 @@
 
 module Network.Sendfile.Types where
 
+#ifdef VERSION_unix
 import qualified Data.ByteString as B
 
 import qualified System.Posix.IO            as P
 import qualified System.Posix.IO.ByteString as PB
 import qualified System.Posix.Types         as P
+#endif
 
 -- |
 --  File range for 'sendfile'.
@@ -16,6 +18,9 @@ data FileRange = EntireFile
                    rangeOffset :: Integer
                  , rangeLength :: Integer
                  }
+
+-- To avoid build error in windows
+#ifdef VERSION_unix
 
 -- | 'openFd's signature changed in @unix-2.8@. This is a portable wrapper.
 openFd :: FilePath -> P.OpenMode -> P.OpenFileFlags -> IO P.Fd
@@ -31,4 +36,6 @@ openFdBS :: B.ByteString -> P.OpenMode -> P.OpenFileFlags -> IO P.Fd
 openFdBS path mode flags = PB.openFd path mode flags
 #else
 openFdBS path mode flags = PB.openFd path mode Nothing flags
+#endif
+
 #endif

